src/components/ArticleList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import ArticlePreview from "./ArticlePreview";
+import Loading from "./Loading";
+import "./style/articles.css";
+
+export default function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetch("https://nc-news-daniel.herokuapp.com/api/articles")
+      .then((response) => response.json())
+      .then((data) => {
+        setIsLoading(false);
+        setArticles(data.articles);
+      });
+  }, []);
```

Comment:
You'll find it easier later on if you use Axios, instead of fetch. But upto you.

I'd also refactor these api requests to be in a single file, so all your requests are together